XenAdmin/ConsoleView/RdpClient.cs

```diff
@@ -211,17 +179,17 @@ public void UpdateDisplay(int width, int height, Point locationOffset)
 
         private bool Connected
         {
-            get { return rdpControl == null ? false : (rdpClient9 == null ? rdpClient6.Connected == 1 : rdpClient9.Connected == 1); }
+            get { return rdpControl == null ? false : rdpClient9.Connected == 1; }

```

Comment:
Style: this could be better written as `rdpControl != null && rdpClient9.Connected == 1`